docs/articles_en/openvino-workflow/running-inference/inference-devices-and-modes/cpu-device/performance-hint-and-threads-scheduling.rst

```diff
@@ -0,0 +1,159 @@
+.. {#openvino_docs_OV_UG_supported_plugins_CPU_Hints_Threading}
+
+Performance Hint and Threads Scheduling 
+=======================================
+
+.. meta::
+   :description: The Threads Scheduling of CPU plugin in OpenVINO™ Runtime
+                 detects CPU architecture and set low-level properties based
+                 on perforamnce hints automatically.
+                 
+Introduction
+############
+
+Even though all supported devices in OpenVINO™ support low-level performance settings, these settings are not recommended for wide useage unless the application is targeted for specific platforms and models. The recommended approach is to configure performance in OpenVINO Runtime is using high level performance hint property ``ov::hint::performance_mode``. Using the perforamnce hints will ensure the best portability and scability of the applications for various platforms and models.
```

Comment:
"The recommended approach is to configure performance" -> "The recommended approach for performance configuration"

Reply:
updated